components/server/src/projects/projects-service.ts

```diff
@@ -111,28 +111,19 @@ export class ProjectsService {
             appInstallationId
         });
         await this.projectDB.storeProject(project);
-        await this.onDidCreateProject(project);
+        await this.onDidCreateProject(project, user);
         return project;
     }
 
-    protected async onDidCreateProject(project: Project) {
+    protected async onDidCreateProject(project: Project, user: User) {
         let { userId, teamId, cloneUrl } = project;
         const parsedUrl = RepoURL.parseRepoUrl(project.cloneUrl);
         if ("gitlab.com" === parsedUrl?.host) {
             const repositoryService = this.hostContextProvider.get(parsedUrl?.host)?.services?.repositoryService;
             if (repositoryService) {
-                if (teamId) {
-                    const owner = (await this.teamDB.findMembersByTeam(teamId)).find(m => m.role === "owner");
-                    userId = owner?.userId;
-                }
-                const user = userId && await this.userDB.findUserById(userId);
-                if (user) {
-                    if (await repositoryService.canInstallAutomatedPrebuilds(user, cloneUrl)) {
-                        log.info("Update prebuild installation for project.", { cloneUrl, teamId, userId });
-                        await repositoryService.installAutomatedPrebuilds(user, cloneUrl);
-                    }
-                } else {
-                    log.error("Cannot find user for project.", { cloneUrl })
+                if (await repositoryService.canInstallAutomatedPrebuilds(user, cloneUrl)) {
```

Comment:
We could also add a comment here that we expect `.canInstallAutomatedPrebuilds()` to be always true for GitLab (and Bitbucket?), because earlier in the Project creation flow, we only show you repositories where you actually have the necessary access rights to install a webhook.